### DISCUSSION TITLE
Verification failing repeatly in campus

### DISCUSSION THREAD

**Discussion by Rosemoe** [community/community#52320] [GitHub Education]
I've provided all I can to prove my enrollment status: 
* dated student ID pamphlet (unfortunately without the school name on register page and I was unable to take several photos of the item)
* dated schedule of my courses this term
* dated digital certificate of enrollment newly exported from school management system today
* school access card
* dated national online verification report of my student status (same as the one in #50543 )
* acceptance material with admission date (last year) and academic system (four years)

I'm unable to provide more materials. But it kept telling me to use another one. Also it was **always** telling me to update my GitHub profile with my full name but when submitting the application I've already set my profile name and bio with my name and school information above though I don't want to do so as this exposes my personal information.
Another problem is that I can not connect to github due to restrictions, which forces me to use some network service and my location is probably wrongly probed. So I was also told to provide remote learning material but I'm not remote learning actually. And even when I used no VPN and connected to my mobile network(but IP recognized to be in JP or HK) or school network(IP is correct), I still got the same tip.

It's very disappointing and annoying.
Now I do not what to do to reverify my student status. What should I do? And, will the failure of reverification interrupt my GitHub access excluding the Student Developer Pack?

**Comment by samus-aran** [comment]
Hi @Rosemoe 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

We provide multiple ways to submit evidence whilst using multiple factors to help assess the authenticity of those pieces of evidence comparing it to the user submitting them. We can only assist users to improve their application as we do not permit manual reviews. If the following tips do not help you progress fully then we would also recommend you reach out to fellow students attending your school how they were successful ⭐ 

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
- In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program. You will have the most success applying if you are able to do so from your school’s campus.
- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
  - If you are using a VPN, please disable it when you reapply. 
  - If you are not using a VPN, try reapplying when connected to the internet from a different network.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
